src/components/PopoverWithMeasuredContent.js

```diff
@@ -53,14 +63,14 @@ class PopoverWithMeasuredContent extends Component {
     constructor(props) {
         super(props);
 
+        this.popoverWidth = this.props.popoverDimensions.width;
+        this.popoverHeight = this.props.popoverDimensions.height;
```

Comment:
What if someone passed undefined to it `popoverDimensions` , So let's use lodashGet if that does not affect performance.